vowpalwabbit/core/include/vw/core/cost_sensitive.h

```diff
@@ -19,32 +20,34 @@ namespace VW
 class cs_class
 {
 public:
-  float x;
-  uint32_t class_index;
-  float partial_prediction;  // a partial prediction: new!
-  float wap_value;           // used for wap to store values derived from costs
+  float x{};
+  uint32_t class_index{};
+  float partial_prediction{};  // a partial prediction: new!
+  float wap_value{};           // used for wap to store values derived from costs
 
   cs_class(float x, uint32_t class_index, float partial_prediction, float wap_value)
       : x(x), class_index(class_index), partial_prediction(partial_prediction), wap_value(wap_value)
   {
   }
-  cs_class() : x(0.f), class_index(0), partial_prediction(0.f), wap_value(0.f) {}
+  cs_class() = default;
 
   bool operator==(const cs_class& j) const { return class_index == j.class_index; }
 };
 class cs_label
 {
 public:
   std::vector<cs_class> costs;
+
+  VW_ATTR(nodiscard) bool is_test_label() const;
+  void reset_to_default();
 };
 
 extern VW::label_parser cs_label_parser_global;
 
 bool is_cs_example_header(const VW::example& ec);
-void default_cs_label(cs_label& ld);
 namespace details
 {
-void output_cs_example(VW::workspace& all, const VW::example& ec);
+// void output_cs_example(VW::workspace& all, const VW::example& ec);

```

Comment:
this on purpose?

Reply:
Unrelated change got mixed in, ill remove that bit